platformio.ini

```diff
@@ -16,7 +16,7 @@
 
 [common]
 platform = espressif32
-platform_packages = framework-arduinoespressif32 @ https://github.com/espressif/arduino-esp32.git
+platform_packages = framework-arduinoespressif32 @ https://github.com/espressif/arduino-esp32.git#46d5afb17fb91965632dc5fef237117e1fe947fc
```

Comment:
Suggest updating this to:

```
platform_packages = framework-arduinoespressif32 @ https://github.com/espressif/arduino-esp32.git#1.0.6
```

(It's the tag for the same hash/release - just a tad more at-a-glance understandable).

BTW, I've verified that this fixes #189. Feel free to add a `Resolves #189` to your PR desc to auto-close upon merge.

Reply:
OK, done.